src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -578,6 +599,7 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
     char in[UCT_IB_MLX5DV_ST_SZ_BYTES(query_hca_cap_in)]   = {};
     struct mlx5dv_context_attr dv_attr = {};
     ucs_status_t status = UCS_OK;
+    uint8_t lag_state = 0;
```

Comment:
indent on column
pls see codestyle build result

Reply:
@yosefe About "pls see codestyle build result", I saw all checks passed, could you please show the link of the failed codestyle build result of this? Thanks!

Reply:
https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=13186&view=logs&jobId=cc064a77-22b5-56bf-ecc0-70b5fe764261

Reply:
some of the errors there can be ignored, but others are justified

Reply:
@yosefe Do you mean the "bash" section in this link? I also saw it, but the "+" part in it is not the real code alignment. Seems it parsed incorrectly. For example in the below code, I didn't add so many spaces after "lag_state".
>>>>>>>>>>>>>>>>>>>>
2021-02-02T15:13:35.8317671Z      ucs_status_t status = UCS_OK;
2021-02-02T15:13:35.8318397Z -    uint8_t lag_state = 0;
2021-02-02T15:13:35.8318951Z +    uint8_t lag_state                                      = 0;
2021-02-02T15:13:35.8319782Z      struct ibv_context *ctx;
2021-02-02T15:13:35.8320274Z      uct_ib_device_t *dev;
<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

Reply:
it should be aligned with lines 599-601 so that "=" is on the same column

Reply:
@yosefe Done. Just wanted to say those "codestyle build result" are not so meaningful.